frontend/source/components/EventCreationComponent/EventCreationComponent.css

```diff
@@ -70,19 +102,21 @@ label {
 }
 
 #blue-header {
-  background-color: rgb(14, 155, 243);
-  padding: 5px; /* Adds space inside the div on all sides */
+  width: 95%;
+  background-color: #007bff;
   color: white;
-  width: 100%;
-  text-align: center; /* Centers horizontally */
+  padding: 20px;
+  text-align: center;
+  border-radius: 8px 8px 0 0;
 }
 
 #event-creation-body {
   flex-grow: 1; /* Allow this section to expand and fill available space */
-  width: 85%;
+  width: 95%; 
   background-color: white;
-  padding: 30px; /* Add space inside the body */
-  margin-top: 20px;
+  padding: 20px; /* Add space inside the body */
+  margin: 10px 0;
+  border-radius: 8px 8px 0 0;
```

Comment:
these changes do a good job improving the look of my EventCreationComponent UI, thanks